### DISCUSSION TITLE
creating cloud optimized geotiff (cog)

### DISCUSSION THREAD

**Discussion by tco95ttocs** [ImageMagick/ImageMagick#5125] [Help]
Hi guys,
is it possible to create a cog with magick.net?
When yes, how is it possible?

Is it possible to create via image.format like jpg or png etc.?

my code is like the following:

MagickImageCollection images = new MagickImageCollection()
images.add(image_xyz); (add some images here)
IMagickImage image = images.Combine(ColorSpace.sRGB)
image.format = MagickFormat.tif   (here is the point where i create serveral formats, the possible plce for the cog!!!))

Thx for helping :)

**Comment by snibgo** [comment]
From https://en.wikipedia.org/wiki/GeoTIFF , this seems to be ordinary TIFF files with extra metadata. IM can create TIFF files, but won't create that metadata, or do anything special if that metadata is in an input file. When reading a file from the net, IM won't use the feature "that clients can predict which range of bytes they need to download."

Exiftool may be useful for you.

**Reply by tco95ttocs** [reply]
thx for the reply...thought there would be the possibility to modify that metadata with IM
